app/build.gradle

```diff
@@ -288,6 +288,12 @@ dependencies {
     implementation 'androidx.activity:activity-compose:1.7.2'
     implementation 'androidx.constraintlayout:constraintlayout-compose:1.0.1'
     implementation "com.google.accompanist:accompanist-systemuicontroller:0.30.1"
+    implementation "androidx.lifecycle:lifecycle-runtime-compose:2.6.2"
+
+    // Coroutines
+    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-android:1.8.0-RC"
+    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-rx2:1.8.0-RC"
+    androidTestImplementation "org.jetbrains.kotlinx:kotlinx-coroutines-test:1.8.0-RC"
```

Comment:
Should we be using RC releases or can we fall back on the latest stable branch stuff?

Reply:
Lates stable for merging